src/components/Icon360.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { IconProps } from './types'
+const Icon360: React.FC<IconProps> = (props: IconProps): any => (
```

Comment:
Could we add an additonal empty line between the imports and the component declaration?